src/clipboard.c

```diff
@@ -2916,11 +2898,16 @@ clip_init_wayland(void)
 		clip_wl.primary.manager = NULL;
 	    }
 	}
+	if (clip_wl.primary.manager == NULL)
```

Comment:
```suggestion
	if (clip_wl.primary.manager == NULL || clip_wl.regular.available)
```
and then get rid of the goto?